src/main/webapp/enrichers/issueStateEnricher.js

```diff
@@ -0,0 +1,72 @@
+import { MessageEnricherBase } from 'symphony-integration-commons';
+import AssignUserService from '../services/AssignUserService';
+import CommentService from '../services/CommentService';
+
+const actions = require('../templates/actions.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+
+const name = 'issueState-renderer';
+const messageEvents = ['com.symphony.integration.jira.event.v2.state'];
+
+export default class IssueStateEnricher extends MessageEnricherBase {
+  constructor() {
+    super(name, messageEvents);
+    this.implements = ['enrich', 'action', 'selected', 'changed'];
+
+    const assignUserService = new AssignUserService(name);
+    const commentService = new CommentService(name);
+
+    this.services = {
```

Comment:
Those variables are a little bit confusing. It's better to write some comments to explain this or leave only the properties that are already being used